core/eolearn/tests/test_eodata.py

```diff
@@ -167,6 +167,41 @@ def test_get_feature():
     assert np.array_equal(eop_bands, bands), 'Data numpy array not returned properly'
 
 
+def test_shallow_copy(test_eopatch_path):
```

Comment:
If we don't have a fixture for `test_eopatch` (that also loads it) yet, it should be added

Reply:
You are right, I realized we already have this fixture. Fixed.